src/DynamicData.Tests/Utilities/ObservableSpy.cs

```diff
@@ -11,6 +12,13 @@ internal static class ObservableSpy
 {
     private static readonly string ChangeSetEntrySpacing = Environment.NewLine + "\t";
 
+    private static class NativeMethods
+    {
+        [DllImport("kernel32.dll", CharSet = CharSet.Unicode, EntryPoint = "OutputDebugStringW")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Interoperability", "SYSLIB1054:Use 'LibraryImportAttribute' instead of 'DllImportAttribute' to generate P/Invoke marshalling code at compile time", Justification = "Affects operation")]
+        public static extern void OutputDebugString(string lpOutputString);
```

Comment:
One issue with using native methods is it would prevent us from promoting this to the main lib.

Reply:
Per the discussion on Slack, I'm okay with that.  This is something to only be used with the test project.  If we move another to the main lib, it would be the base functions, and not any overloads that use this.

I wouldn't even suggest checking in code that enables this, it is just a temporary for situations where an issue won't repro in debug builds.